src/app/styl/views/loading.styl

```diff
@@ -20,6 +20,7 @@
         background-position-x 50%
         background-position-y 0px
         background-size cover
+        -webkit-filter: blur(15px) brightness(0.7)

```

Comment:
This is the only thing I'm not sure at all, without blur the transition is abruptly but on the other hand is cool to see without blur too...